vtex-apps/slider-layout
# Use 'vtex.responsive-values' to handle values passed to itemsPerPage prop

#### What problem is this solving?

The `slider-layout` used its own logic to handle the changes in display size and device. This resulted in duplicated logic since we already have a solution for this in the form of `vtex.responsive-values`. With this PR we're cleaning things up by removing that unnecessary logic.

Also, with the most recent release of `vtex.responsive-values`, the support for media-queries was added, so we now support it in the `slider-layout` as well.

#### How to test it?



Go to [Workspace](https://victormiranda--storecomponents.myvtex.com/), and the sliders should be working fine. This is more of an internal change, so what you should be aware of is any strange behavior.

#### Related to / Depends on

This was motivated by https://github.com/vtex-apps/responsive-values/pull/9.

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/3Asm9mqAvPJsI/giphy.gif)
